File: WinFormsAppProyecto/Modelos/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Modelos
{
    public abstract class Usuario
    {
        public int id {  get; set; }
        public string nombre_usuario { get; set; }
        public string nombre_real { get; set; }
        public string fecha_nacimiento { get; set; }
        public string email { get; set; }
        public string password { get; set; }

    }
}

File: AndroidProyecto/app/src/main/java/com/example/androidappproyecto/data/data/modelos/Usuario.kt
package com.example.androidappproyecto.data.data.modelos

import java.io.Serializable

open class Usuario(
    open val id: Int = 0,
    open val nombre_usuario: String,
    open val nombre_real: String,
    open val fecha_nacimiento: String? = null,
    open val email: String,
    open val password: String
) : Serializable

